tests/extension/modules_test.php

```diff
@@ -89,21 +95,7 @@ public function test_get_module_infos()
 			), $acp_modules);
 
 		// Find a specific module info file (mcp_a_module)
-		$this->acp_modules->module_class = 'mcp';
-		$acp_modules = $this->acp_modules->get_module_infos('mcp_a_module');
-		$this->assertEquals(array(
```

Comment:
Nope, it does not work anymore, because there is no [module_manager::module_class](https://github.com/phpbb/phpbb/pull/3552/files#diff-9b05e985bd4f61bb1bf4c1893a24a624L559) member var. However, IMO this means that I have to update the get_module_infos() params in some way, or would it be enough to just throw an exception when module class is not definded? Or just default to something?